Source/WebKit/UIProcess/Extensions/WebExtensionAction.h

```diff
@@ -91,9 +92,9 @@ class WebExtensionAction : public API::ObjectImpl<API::Object::Type::WebExtensio
     void propertiesDidChange();
 
     CocoaImage *icon(CGSize);
```

Comment:
This should change to match `WebExtension`. It would simplify that implementation a bit.
```suggestion
    RefPtr<WebCore::Icon> icon(WebCore::FloatSize idealSize);
```